### ISSUE TITLE
Error: type "image" is not valid for variant

### ISSUE THREAD

**Issue by cryptotester** [ProtonProtocol/nft-mint#3]
This error can be fixed with:

```
const SCHEMA = {
    series: "uint16",
    image: "image",
    name: "string"
}
```

Replace:

```
    image: "image",
```

With:

```
    image: "string",
```

Sorry that I don't do the PRs, I'm really into coding but I like to contribute also when I find issues so it may help others. Cheers

**Comment by jafri** [comment]
What is the difference here?

Is it some sort of weird formatting thing

On Mar 27, 2021, at 10:53 AM, cryptotester ***@***.***> wrote:

﻿
[△EXTERNAL]



This error can be fixed with:

const SCHEMA = {
    series: "uint16",
    image: "image",
    name: "string"
}


Replace:

    image: "image",


With:

    image: "image",


Sorry that I don't do the PRs, I'm really into coding but I like to contribute also when I find issues so it may help others. Cheers

**Comment by cryptotester** [comment]
@jafri I edited the post (before you answered but I guess you received it via email and so could see the first version only). Check it now. Replace image with string. Tested and it works.

**Comment by jafri** [comment]
Resolved in 5016e674e6aef4ee228860a7cde7b5aaea101b10
